include/ts/apidefs.h.in

```diff
@@ -430,6 +430,7 @@ typedef enum {
   TS_EVENT_SSL_SESSION_NEW                      = 2001,
   TS_EVENT_SSL_SESSION_REMOVE                   = 2002,
   TS_AIO_EVENT_DONE                             = 3900,
+  TS_EVENT_AIO_DONE                             = TS_AIO_EVENT_DONE,
```

Comment:
This is a good fix, but I'd like to suggest something a little more invasive here:

1. Break this particular fix into a separate PR, and add comments that TS_AIO_EVENT_DONE is deprecated in favor of TS_EVENT_AIO_DONE. Then mark this PR for an 8.1.0 back port, so we get this into 8.x (but to be backwards compatible, we need both the old and the new define).

2. On master (in this PR if so suited), remove TS_AIO_EVENT_DONE. Meaning, it'll be gone for 9.0.0.